Create Payouts page according issue 1070 (#1086)
btcpayserver/btcpayserver-doc: docs/Payouts.md

# Payouts

Tied into the [Pull Payments](./PullPayments.md) we have the **payouts** tab. 

> In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals. 

Reviewer: ```suggestion
This feature allows you to process pull payment (refunds, salary payouts, or withdrawals). 
```

> ## How does it work? 

Reviewer: Doesn't explain how payouts work cc @Kukks can you help us with defining this?

We will go through two examples, one will be a Refund, and the other will be a salary payout. 

> As of writing this guide (v1.4.7), there is no automation for the last example. 

Reviewer: Not relevant yet.
```suggestion
```

### Example

> Let's start with the Refund example. 

Reviewer: ```suggestion
Let's start with the refund example. 
```

> The Customer has bought an item in your store but sadly has to return the item. He wants a refund. 

Reviewer: ```suggestion
The customer has bought an item in your store but sadly has to return the item. They want a refund. 
```

> Within BTCPay, the merchant creates a [Refund](./Refund.md) and provides the Customer with the appropriate link to claim his funds. 

Reviewer: ```suggestion
Within BTCPay, you can create a [Refund](./Refund.md) and provide the customer with the link to claim their funds. 
```

> Whenever the Customer has given his address and claimed the funds, the merchant now has an entry in Payouts. 

Reviewer: ```suggestion
Whenever the customer has given their address and claimed the funds, it will be shown in the `Payouts`. 
```
